app/jobs/assignment_repo/create_github_repository_job.rb

```diff
@@ -56,12 +49,29 @@ def perform(assignment, user)
       duration_in_millseconds = (Time.zone.now - start) * 1_000
       GitHubClassroom.statsd.timing("exercise_repo.create.time", duration_in_millseconds)
 
-      PorterStatusJob.perform_later(assignment_repo, user)
+      if assignment.starter_code?
+        assignment.invitation&.importing_starter_code!
+        ActionCable.server.broadcast(
+          RepositoryCreationStatusChannel.channel(user_id: user.id),
+          text: IMPORT_STARTER_CODE,
+          status: assignment.invitation&.status
+        )
+        PorterStatusJob.perform_later(assignment_repo, user)
+      else
+        assignment.invitation&.completed!
+        ActionCable.server.broadcast(
+          RepositoryCreationStatusChannel.channel(user_id: user.id),
+          text: Creator::REPOSITORY_CREATION_COMPLETE,
+          status: assignment.invitation&.status
+        )
```

Comment:
Should we be recording stats about successful repo creation on this vs the old way? if so using  `GitHubClassroom.statsd`?